src/main/java/com/napier/sem/DAO.java

```diff
@@ -137,16 +135,18 @@ public DAO(Connection connection)
                 break;
         }
 
-        // Define the SQL query as a string
-        String statementString = "SELECT code, country.name, continent, region, country.population, city.name AS capital\n" +
-                "FROM country\n" +
-                "    JOIN city ON country.capital = city.id\n" +
-                whereClause + 
-                "ORDER BY country.population DESC";
+            // Define the SQL query as a string
+            String statementString = "SELECT code, country.name, continent, region, country.population, city.name AS capital\n" +
+                    "FROM country\n" +
+                    "    JOIN city ON country.capital = city.id\n" +
+                    whereClause +
+                    "ORDER BY country.population DESC";
+
+            return ExecuteCountryStatement(statementString);
 
-        return ExecuteCountryStatement(statementString);
```

Comment:
i'm not sure why any of this has changed, could we revert it? i think the indentation is wrong now.

Reply:
Yes Sorry

Reply:
will review

Reply:
you should be able to revert to a previous commit in the git menu

Reply:
I can't actually see it as being removed in the DAO class file